routes/skills.py

```
@@ -1,260 +1,362 @@
-
+        # Get the skill service instance
+        skill_service = get_skill_service()
+
+        skill = skill_service.get_skill_by_id(skill_id, include_relationships=True)
+        if not skill:
+            return jsonify({'error': 'Skill not found'}), 404
+        
+        variants = [{'id': v.id, 'name': v.variant_name} for v in skill.variants]
+        return jsonify(variants)
+        
     except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)